_includes/pillars.html

```diff
@@ -0,0 +1,32 @@
+<div class="row">
+    <div class="col-md-8 col-md-offset-2 text-center">
+      <p><b>MINISTRY OBJECTIVE</b></p>
```

Comment:
We don't use uppercase anywhere else in our website. Please change to title case.

Reply:
Even when the [design ](https://www.figma.com/file/NB9fjygFVzs3GfyUq0Xivn/Website?node-id=407%3A56&t=0xbZp9J3jn5z3erj-0) calls for it? Sorry I'm just not aware how stringent this design principle is.
We also do have all caps for our tab menus (though idk if that really counts)

Reply:
Yes, even when the design calls for it. The nav bar is the only place so it's an exception -- mainly to keep consistent with the logo which is in all caps. Unless we do a redesign where we incorporate uppercase text, I would keep it out.

Reply:
@jennxhong Thanks for being the liaison between the earlier designs and newer changes. Would love to meet at some point about the design system that's been built out so I can work with it and know its capabilities and limitations. Definitely want to design systematically instead of creating one-off styles - thanks for pushing for best practice.

@jyc1264 I don't want this one element to be a blocker from shipping the site. We can use title case for now per @jennxhong 's recommendation. We can discuss uppercase usages at a future time. 

Thank you both for the implementations and reviews!

Reply:
updated all caps cases to be title case